GravatarAppTests/Helpers/URLSessionProfileMock.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 @testable import GravatarApp
 import GravatarUI
 
-final class URLSessionAvatarPickerMock: URLSessionProtocol {
+final class URLSessionProfileMock: URLSessionProtocol {
```

Comment:
This is also extensively used by the AvatarPicker view model, so maybe a more generic name would be good.
Maybe just `URLSessionMock` is good enough?

There's also a warning here but it should be solved with a merge from trunk

Reply:
Yeah, since we have an specific "Profile" section separated from "AvatarPicker" section, the first thought that came to my mind was "Profile-section-specific URL Session". It didn't occur to me to think of Profile as global in this context.

Thank you for the update 🙏